pandas/io/excel/_openpyxl.py

```diff
@@ -410,6 +438,17 @@ def _convert_to_protection(cls, protection_dict):
 
         return Protection(**protection_dict)
 
+    @staticmethod
+    def _convert_to_excel_format(row, col):
+        col_name = ""
+
+        while col > 0:
+            mod = (col - 1) % 26
+            col_name = str(chr(65 + mod)) + col_name
+            col = (col - mod) // 26
+
+        return "%s%d" % (col_name, row)
+
```

Comment:
Due to the nature of how merging cells in `write_only` mode works, we need the row and column in Excel notation rather than its index. This converts a row and column number to excel format (e.g. A1, ZZ99, etc.).